StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFlowController.swift

```diff
@@ -22,6 +22,19 @@ extension PaymentSheet {
         case saved(paymentMethod: STPPaymentMethod)
         case new(confirmParams: IntentConfirmParams)
         case link(option: LinkConfirmOption)
+
+        var paymentMethodTypeAnalyticsValue: String? {
```

Comment:
Did we consider using `PaymentSheet.PaymentMethodType` instead of exposing a String?
Then for the saved case, we can have `return paymentMethod.paymentSheetPaymentMethodType()`

Reply:
Yes this was explored, 2 reasons why I didn't go this route:

1. There is no representation for Apple Pay in `PaymentSheet.PaymentMethodType`. I don't think we should add one just for analytics. I thought about using `.dynamic("apple_pay")` to fill this void... but that creates an invalid/non-existent payment method type that someone could misuse in the future. 
2. Based on the fear of this being misused, I wanted this string to be scoped strictly for analytics.